src/test/kotlin/org/elm/ide/inspections/ElmIncompletePatternInspectionTest.kt

```diff
@@ -90,6 +90,137 @@ foo it =
             --EOL
 """)
 
+//    fun `test no existing branch in parenthesis`() = checkFixByText("Add missing case branches", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (<error>case{-caret-}</error> it of<error>)</error>
+//""", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (case it of
+//        Bar ->
+//            --EOL
+//
+//        Baz ->
+//            --EOL
+//
+//        Qux ->
+//            --EOL
+//    )
+//""")
+
+//    fun `test one existing branch in parenthesis`() = checkFixByText("Add missing case branches", """
```

Comment:
On `master`, this test fails with this diff:
![image](https://user-images.githubusercontent.com/6695040/103339490-c9b12b00-4a81-11eb-8f46-0c86ce6a626e.png)

Reply:
On my branch, this test fails with this diff:
![image](https://user-images.githubusercontent.com/6695040/103339827-beaaca80-4a82-11eb-97ab-90272e00e78f.png)